.github/workflows/notebook-test.yml

```diff
@@ -50,6 +50,12 @@ jobs:
           files: "docs/**/*.ipynb"
           separator: "\n"
 
+      - name: Get changed dependency files
```

Comment:
Nit: maybe call this "changed config"? We rerun the tests when changing any file in nb-tester, which I think is a good idea to ensure we don't break things.